src/Hl7.Fhir.Specification.Tests/Source/TerminologyTests.cs

```diff
@@ -948,6 +963,59 @@ public async Task<Resource> ResolveByCanonicalUriAsync(string uri)
             public Task<Resource> ResolveByUriAsync(string uri) => throw new NotImplementedException();
         }
 
+        private class OnlyCodeSystemResolver : IAsyncResourceResolver, IConformanceSource
```

Comment:
I do this regularly too (creating test implementations), but shouldn't we use Moq for this?

Reply:
Good point, let's try to use Moq for these kinds of things next time.